ImprovedAnvil/lib/creatures/02_patches_vanilla.tph

```diff
@@ -86,10 +86,26 @@ COPY_EXISTING ~dragblac.cre~ ~override/dragblac.cre~
   ADD_CRE_ITEM ~S!trap~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BELT~
   ADD_CRE_ITEM ~S!dblack~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~GLOVES~
   ADD_CRE_ITEM ~minhp1~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~AMULET~
+  ADD_CRE_ITEM ~s!birchim~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BOOTS~  
```

Comment:
The item you've created is called s!brchim, and you then write "S!birchim" everywhere.

Reply:
the capitalization requirements (and consistency within the mod) are opaque to me.

Reply:
This is not a capitalization issue, it's the fact that you've created an item that's called "S!brchim" (without "i" after "b") and then you try to insert an item called "S!birchim" (note the "i" after "b") into the creature's inventory. Such item does not exist.